.github/workflows/deploy.yml

```diff
@@ -2,7 +2,7 @@ name: Deploy
 
 on:
   push:
-    branches: [main]
+    branches: [main, lit-next]
```

Comment:
Can we remove this change now that we are going to use Lit v2 in the main branch?